src/bp/sdk/botpress.d.ts

```diff
@@ -1389,6 +1390,7 @@ declare module 'botpress/sdk' {
     type?: FlowNodeType
     timeoutNode?: string
     flow?: string
+    isNew?: boolean
```

Comment:
Why that ? I mean you want "new nodes"  to become the standard not something new. We should do the opposite and mark old nodes as legacy, that way we can deprecate them easily instead of polluting our users data with isNew that eventually will do nothing because those node will be the standard

Reply:
or is it to mark that a ui element (node) is newly created ?

Reply:
isNew is only being used to automatically open the right side panel when the node is created, it has no other purpose. It was implemented this way in Albert and it worked well